server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoError/index.ts

```diff
@@ -58,6 +63,17 @@ class NoConnectionAvailableRule extends BaseMongoExceptionRetryRule {
 			(error.message as string).startsWith(NoConnectionAvailableRule.messagePrefix)
 		);
 	}
+
+	public shouldRetry(): boolean {
+		if (this.connectionNotAvailableMode === "stop") {
+			// This logic is to automate the process of handling a pod with death note on it, so
+			// kubernetes would automatically handle the restart process.
+			Lumberjack.warning(`${this.ruleName} will restart the process`);
+			process.exit(1);
```

Comment:
Does this generate a SIGTERM? `process.kill()` seems to allow us to specify `signal`. Should we use `kill` instead?